src/core/test/system/borrower/close.t.sol

```diff
@@ -50,7 +59,8 @@ contract CloseTest is SystemTest {
         uint loanId = borrower.issue(collateralNFT_, tokenId);
         // transfer nft to random user / borrower still loanOwner
         borrower.approveNFT(collateralNFT, address(this));
-        collateralNFT.transferFrom(borrower_, address(this), tokenId);  
+        collateralNFT.transferFrom(borrower_, address(this), tokenId);
```

Comment:
It could be any address here correct? you just want to ensure the loan owner isn't owning the collateral nft.